cnd/Cargo.toml

```diff
@@ -59,21 +59,11 @@ uuid = { version = "0.8", features = ["serde", "v4"] }
 void = "1.0.2"
 warp = { version = "0.2", default-features = false }
 
-# These versions need to be "in sync".
-# web3 0.8 gives us primitive-types 0.3.0
-# primitive-types 0.3.0 with the "rlp" feature gives us "rlp" version 0.4.2
 [dependencies.web3]
```

Comment:
`web3` was only declared separately so I could add this comment. Feel free to declare it next to all the other dependencies now!

Reply:
Cool, fixed.  Thanks